src/app/courses/components/add-course/add-course.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-add-course',
+  templateUrl: './add-course.component.html',
+  styleUrls: ['./add-course.component.scss'],
+})
+export class AddCourseComponent {
+  public courseTitle: string;
+
+  constructor() {}
+
+  public onDurationChange(duration: number): void {
+    console.log('duration: ' + duration);
```

Comment:
as per markup it should be visible on template

Reply:
Yes and it is, please check line 16 of the `input-duration.component.html` file, it is hidden by default, but becomes visible when you start typing.

Reply:
really, I missed it for the first time. Then it is all right